src/backend/distributed/sql/udfs/worker_split_shard_replication_setup/latest.sql

```diff
@@ -1,15 +1,20 @@
+DROP FUNCTION pg_catalog.worker_split_shard_replication_setup(pg_catalog.split_shard_info[]);
+DROP TYPE pg_catalog.replication_slot_info;
+DROP TYPE pg_catalog.split_shard_info;
+
 CREATE TYPE citus.split_shard_info AS (
     source_shard_id bigint,
     distribution_column text,
     child_shard_id bigint,
     shard_min_value text,
     shard_max_value text,
-    node_id integer);
+    node_id integer,
+    operation_id bigint);
 ALTER TYPE citus.split_shard_info SET SCHEMA pg_catalog;
 COMMENT ON TYPE pg_catalog.split_shard_info
     IS 'Stores split child shard information';
 
-CREATE TYPE citus.replication_slot_info AS(node_id integer, slot_owner text, slot_name text);
+CREATE TYPE citus.replication_slot_info AS(node_id integer, slot_owner text, operation_id bigint, slot_name text);
```

Comment:
Instead of adding operation_id to these types, could it instead be added as an argument to the function? No need to return it either I think, because it's already known by the caller.

Reply:
makes sense. I have just done it. Thanks!